File: 2.0/2/A.cpp
#include <iostream>
using namespace std;

int main() {
    int x = 0, mxm = 0, ans = 0;
    while(true) {
        cin >> x;
        if (x == 0) break;
        if (x == mxm) ans++;
        else if (x > mxm) {
            ans = 1;
            mxm = x;
        }
    }

    cout << ans << endl;

    return 0;
}

File: 2.0/2/A.go
package main

import "fmt"

func main() {
	ans := 0
	mxm := 0
	var x int

	for {
		fmt.Scan(&x)
		if x == 0 {
			break
		}
		if x > mxm {
			mxm = x
			ans = 1
		} else if x == mxm {
			ans++
		}
	}

	fmt.Println(ans)
}